### PR TITLE
Add a Reviews section to Code of Conduct

### PR DESCRIPTION
**Pull request by nickmeinhold**
Following up from our RTC today, this is my attempt at capturing what we agreed on.

### CODE REVIEW

File: .contributing/code-of-conduct.md [hunk 1]
```diff
@@ -4,10 +4,10 @@
 
 #### Reviews 
 
-- 1 review is required for merging 
+- Generally only 1 approval is required before merging.  The exceptions are: 
+  - Files in the `.contributing/` folder, where approvals are needed from the whole team before merging
```
Comment (T-Davis) [inline .contributing/code-of-conduct.md:8]: throw('numReviewsNotMetException'); 😄

**Review by T-Davis — APPROVED** [review]
Looks good! Thanks for capturing 👍

File: .contributing/code-of-conduct.md (hunk 1 above)
Comment (nickmeinhold) [reply .contributing/code-of-conduct.md:8]: Haha we need to get you out of the error handling area for an evening or two mate 😄

**Comment by nickmeinhold** [comment]
Thanks @T-Davis!  Sorry for the muck around but, as we discussed today, there was a couple more points I wanted to add to this.  In hindsight I think the appropriate thing would have been to set the PR back to Draft status, to indicate I wanted to make changes before requesting another review (I believe that's very much in line with what the Draft status is meant to be used). Anyway I'll do that now, just wanted to communicate why.  Thanks!

File: .contributing/code-of-conduct.md [hunk 2]
```diff
@@ -2,16 +2,20 @@
...
+- When a PR has approval(s), if someone else wants changes we don't merge until the request is resolved.
 - The reviewer merges when they are happy that everything has been addressed 
-- Please use "Changes requested" to hand back control until all questions have been resolved (avoids the ambiguous state of unanswered questions in an approved PR) 
+- Please use "Changes requested" to hand back control until all questions have been resolved (in order to avoid the ambiguous state of unanswered questions in an approved PR) 
+- All merges onto `dev` should be squashed (with the squash & merge option at merge time) 
 
 #### Treat PRs as the start of a process, not the end 
 
 - When you finish a PR you have setup the next stage, which is a collaborative process. 
+- Questions and discussion should be welcomed as an attempt to improve our project. If you feel frustrated, try to get to the bottom of it and consider having a conversation with the person involved.
```
Comment (tamari-gray) [inline .contributing/code-of-conduct.md:18]: 😍

**Review by tamari-gray — APPROVED** [review]
lgtm!

**Review by T-Davis — APPROVED** [review]
Getting better all the time 👍

**Pull request merged**